neonvm/controllers/virtualmachine_controller.go

```diff
@@ -194,7 +190,7 @@ func (r *VirtualMachineReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 }
 
 // finalizeVirtualMachine will perform the required operations before delete the CR.
```

Comment:
```suggestion
// doFinalizeVirtualMachine will perform the required operations before delete the CR.
```

Reply:
should be `doFinalizerOperationsForVirtualMachine`, no?

Reply:
ah right, my bad